packages/server/tests/integration/v1/roles.spec.ts

```diff
@@ -27,40 +26,8 @@ describe("GET /api/v1/roles", () => {
 
     const { user } = await createUser();
 
-    const roleId = uuid();
-
-    await database.transaction(async (trx) => {
-      await trx
-        .insert({
-          id: roleId,
-          name: "CreateRole",
-        })
-        .into("roles");
-
-      const readRolePerms = await trx
-        .select("id")
-        .from("permissions")
-        .where({
-          type: "role",
-          action: "read",
-        })
-        .first();
-
-      await trx
-        .insert({
-          id: uuid(),
-          role_id: roleId,
-          permission_id: readRolePerms.id,
-        })
-        .into("permissions_roles");
-
-      await trx
-        .insert({
-          id: uuid(),
-          role_id: roleId,
-          user_id: user.userId,
-        })
-        .into("roles_users");
+    await createRoleWithPermissions(user.userId, ["role:read"], {
+      roleName: "CreateRole",
```

Comment:
We're assigning the `role:read` permission can you give an appropriate name accordingly.